File: CH3-desicion-sructure-and-boolean-logic/EX3-mass-and-height/source-code/massAndHeight.cpp
#include <iostream>
using namespace std;

int main()
{
  float mass;
  float weight;

  cout << "Enter a object's mass";
  cin >> mass;

  weight = mass * 9.8;

  if (weight > 1000)
  {
    cout << "It is too Heavy";
  }

  if (weight < 10)
  {
    cout << "It is too light";
  }
}

File: CH3-desicion-sructure-and-boolean-logic/EX3-mass-and-height/massAndHeight.java

import java.util.Scanner;

public class massAndHeight {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    double mass;
    double weight;

    System.out.print("Enter a object's mass :");
    mass = input.nextFloat();

    weight = mass * 9.8;

    if (weight > 1000) {
      System.out.println("It is too Heavy");
    }
    if (weight < 10) {
      System.out.println("It is too light");
    }
    input.close();
  }
}